config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  get 'events/new'
+  get 'events/index'
   devise_for :users, :controllers => { registrations: 'registrations'}
   resources :users
+  resources :events
```

Comment:
**`OPTIONAL`**
Routes can be restricted to this `resources :events, only: [:new, :create, :show, :index]`